src/terrain/terrain_generation/worley_noise.cpp

```diff
@@ -0,0 +1,81 @@
+#include "worley_noise.hpp"
+
+namespace terrain {
+
+namespace terrain_generation {
+
+WorleyNoise::WorleyNoise(double tile_size = 1) : tile_size_(tile_size) {}
+
+[[nodiscard]] inline double
+WorleyNoise::distance(double x, double y, WorleyPoint point) {
+    return std::sqrt(
+        std::pow(x - point.x_position, 2) + std::pow(y - point.y_position, 2)
+    );
+}
```

Comment:
you
can't
inline
in
a
C++ 
file

Reply:
I just coppied and paseted from hpp, then forgot.